src/services/apimPolicyBuilder.ts

```diff
@@ -132,7 +139,7 @@ export class ApimPolicyBuilder {
       }
     });
 
-    const elements = {
+    const policy = {
       $: attributes,
       "openid-config": oidConfig,
       "issuer-signing-keys": signingKeys,
```

Comment:
ah, was wondering about this... was hoping a public key could be specified OR included in the JWT token.

Reply:
Yes, you can manually set your own signing / decryption keys in the yaml config.  Support full configuration from APIM docs @ https://docs.microsoft.com/en-us/azure/api-management/api-management-access-restriction-policies#ValidateJWT